workitem/workitem_repository_blackbox_test.go

```diff
@@ -769,3 +769,13 @@ func (s *workItemRepoBlackBoxTest) TestList() {
 
 	})
 }
+
+func (s *workItemRepoBlackBoxTest) TestDeleteWorkitem() {
```

Comment:
We create a `workitem revision` entry after deleting a workitem. That should also be checked.

Reply:
@jarifibrahim we already have a test to check the revision entry after deleting a workitem - https://github.com/fabric8-services/fabric8-wit/blob/master/workitem/workitem_revision_repository_blackbox_test.go#L51

Reply:
Thank you @DhritiShikhar. I didn't see that before :)